servers/rendering/renderer_viewport.h

```diff
@@ -208,6 +208,7 @@ class RendererViewport {
 	Vector<Viewport *> _sort_active_viewports();
 	void _viewport_set_size(Viewport *p_viewport, int p_width, int p_height, uint32_t p_view_count);
 	bool _viewport_requires_motion_vectors(Viewport *p_viewport);
+	void _viewport_set_force_motion_vectors(Viewport *p_viewport, bool p_force_motion_vectors);
```

Comment:
Just like the other `_viewport_` functions, this takes a `Viewport *` as the first argument, rather than RID